tools/cldr-code/src/main/java/org/unicode/cldr/test/ExampleGenerator.java

```diff
@@ -3027,6 +3044,47 @@ void handleQuarters(XPathParts parts, String value, List<String> examples) {
         examples.add(sdf.format(sample));
     }
 
+    /* Add relative date/time examples, choosing appropriate
+     * patterns as needed for relative dates vs relative times.
+     * Additionally, for relativeTimePattern items, ensure that
+     * numeric example corresponds to the count represented by the item.
+     */
+    private void handleRelative(
+            String xpath, XPathParts parts, String value, List<String> examples) {
+        String skeleton;
+        String type = parts.findAttributeValue("field", "type");
+        if (type.startsWith("hour")) {
+            skeleton = "Hm";
+        } else if (type.startsWith("minute") || type.startsWith("second")) {
+            skeleton = "ms";
+        } else if (type.startsWith("year")
+                || type.startsWith("month")
+                || type.startsWith("quarter")) {
+            skeleton = "yMMMM";
+        } else {
+            skeleton = "MMMMd";
+        }
+        String checkPath =
+                "//ldml/dates/calendars/calendar[@type=\"gregorian\"]/dateTimeFormats/availableFormats/dateFormatItem[@id=\""
+                        + skeleton
+                        + "\"]";
+        String dateFormat = cldrFile.getWinningValue(checkPath);
+        SimpleDateFormat sdf = icuServiceBuilder.getDateFormat("gregorian", dateFormat);
+        String sampleDate = sdf.format(DATE_SAMPLE);
+        String example1 =
+                value.substring(0, 1).toUpperCase() + value.substring(1) + " (" + sampleDate + ")";
```

Comment:
Hmm. We need to use locale-sensitive Titlecasing (not just toUpperCase the first char), and we also only want to do this if the contextTransforms data says that relative fields should be title-cased in a stand-alone context. Again, I will fix this in [CLDR-18025](https://unicode-org.atlassian.net/browse/CLDR-18025) "Further work on examples for relative dates and times".